src/components/header/_header.scss

```diff
@@ -176,6 +178,20 @@
     }
   }
 
+  &__org-logo--multi {
```

Comment:
Yeah I was just trying to be reusable as possible. Having had a play with it I can see this is probably our best way of doing it. Although I think we can remove the media query because as far as I can see its not doing much apart from slightly reducing the column gap

Reply:
the difference is pretty minimal so happy to remove the mq code  
![image](https://github.com/ONSdigital/design-system/assets/149602681/4e521682-bb24-40ab-9060-546ffb277de6)
![image](https://github.com/ONSdigital/design-system/assets/149602681/5d6ebfe6-9497-4085-bca9-0e719fec7ced)